plugin/forward/dnstap.go

```diff
@@ -48,9 +47,8 @@ func toDnstap(f *Forward, host string, state request.Request, opts options, repl
 	r := new(tap.Message)
```

Comment:
lets allocate the `r` object inside the `if` section below

Reply:
done